scripts/aio_cleanup.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+import datetime
+import dateutil.parser
+import os
+import requests
+
+from novaclient import client
+
+JENKINS_API_USER = os.environ['JENKINS_API_USER']
+JENKINS_API_PASS = os.environ['JENKINS_API_PASS']
+
+USERNAME = os.environ['OS_USERNAME'],
+API_KEY = os.environ['OS_PASSWORD'],
+TENANT_ID = os.environ['OS_TENANT_NAME'],
+AUTH_URL = os.environ['OS_AUTH_URL'],
+
+
+def jenkins_node(name, slave_names):
+    return name in slave_names
+
+
+def main():
+    nova = client.Client(2, USERNAME, API_KEY, TENANT_ID, AUTH_URL,
+                         region_name="IAD")
```

Comment:
Can we make the region an environment variable or a parameter to the script? I've been creating some servers in DFW since there's more capacity for onmetal2-io there.